### ISSUE TITLE
Count not connect to jitpack for  OneSignal Plugin

### ISSUE THREAD

**Issue by kareemzok** [jitpack/jitpack.io#3983]
Hello,

Suddenly I m getting the following error

`ERROR: Could not GET 'https://jitpack.io/gradle/plugin/com/onesignal/onesignal-gradle-plugin/'. Received status code 522 from server: Origin Connection Time-out`

Also when I m trying to enter  https://jitpack.io/gradle/plugin/com/onesignal/onesignal-gradle-plugin/ I m getting the following:

```
Connection
There seems to be an issue connecting to our servers. Please try again in a few minutes.

```

Any advice, please

**Comment by okwast** [comment]
Having the same issue with `https://jitpack.io/javax/inject/javax.inject/1/javax.inject-1.jar` since 20 minutes or so.

Maybe there are some server issues?

**Comment by kareemzok** [comment]
Yes exactly, I m getting the same. 
What should we do? it happened yesterday and im unable to work since then

**Comment by okwast** [comment]
Not sure if it helps, but I "solved" our issue by changing the order in the `build.gradle`, so the package is used from maven instead of jitpack.

Another approach would be to add `www.` to the url of jitpack. That solved a problem we had with another package.

Hope something helps.

**Comment by kareemzok** [comment]
Thanks for your reply. I lll try that and let you know but i think the order solution tried it before and didnt work with me but i ll recheck again.
Also OneSignal team told me to open issue on their github account. So i ll try that also.

Concerning the order you mean change the calling position right ?

**Comment by okwast** [comment]
Yep, inside the build.gradle inside repositories do the call to add jitpack in the end.

The solution helped because the broken package from jitpack was also available on maven repo.
Therefore changing the order caused the process to check on maven repo first and as it is found there, there is no need to go to the broken jitpack.

Maybe you can remove jitpack altogether.

**Comment by kareemzok** [comment]
Thanks for your help i ll try it tomorow and let you know if its resolved

**Comment by kareemzok** [comment]
Hello @okwast, Your solution with adding the www works. Thanks a lot for your help!
